docs/user-guide/side-bar.md

```diff
@@ -1,94 +1,91 @@
 
-The returned information are viewed in Text plain format by default. In order to change it:
+<img src="../img/side-bar/format-options.jpg" class="ms-docimage" style="max-width:400px;"/>
 
-* **Navigate** into *Settings* from the main menu bar.
+The information will be returned in the format chosen by the user. For exaple with *PROPERTIES* format as follows:
 
-<img src="../img/side-bar/settings.jpg" class="ms-docimage"/>
-
-* **Select** from the list menu *Identify response format* another format (e.g. HTML) then **Click** on a feature.
-
-<img src="../img/side-bar/info_formats.jpg" class="ms-docimage" style="max-width:500px;"/>
-
-* The informations will be returned as HTML accordingly.
-
-<img src="../img/side-bar/html_info.jpg" class="ms-docimage"/>
+<img src="../img/side-bar/format_example.jpg" class="ms-docimage"/>
 
 !!! warning
     This global settings could be overwritten by a layer-specific configuration (see [Feature Info Form](layer-settings.md#feature-info-form)).
 
-Below the point coordinates there are three icons:
 
-* **More Info** <img src="../img/button/more_info_icon.jpg" class="ms-docbutton"/> shows a popup window displaying the point address.
+In addition to the features, the following information is provided by the *Identify Tool*: 
 
-<img src="../img/side-bar/more_info.jpg" class="ms-docimage"/>
+* The **point address**  trought the *More Info* button  <img src="../img/button/more_info_icon.jpg" class="ms-docbutton"/>
 
-* **Coordinates Editor** <img src="../img/button/coordinates_editor_icon.jpg" class="ms-docbutton"/> makes available a tool to perform queries using decimal or aeronautical coordinates (we will explain more in depth this tool in the next paragraph).
+<img src="../img/side-bar/more-info.jpg" class="ms-docimage"/>
 
-* **Highlight Features** <img src="../img/button/highlight_features_icon.jpg" class="ms-docbutton"/> highlights the resulting features.
+* The **coordinates** <img src="../img/button/coordinates_editor_icon.jpg" class="ms-docbutton"/> of the point
 
-<img src="../img/side-bar/highlight_features.jpg" class="ms-docimage"/>
+<img src="../img/side-bar/coordinate.jpg" class="ms-docimage"/>
 
-Using the <img src="../img/button/zoom-layer.jpg" class="ms-docbutton"/> button it is possible to zoom the highlighted feature.
+!!! note
+    The point coordinates are viewed in **decimal** or **areonautical** format. It is possible to change the format by the *setting* button <img src="../img/button/gear_icon.jpg" class="ms-docbutton" style="max-heigth:50px;"/> 
+
+* The **Highlight Features** <img src="../img/button/highlight_features_icon.jpg" class="ms-docbutton"/> highlights the resulting features. Using the <img src="../img/button/zoom-layer.jpg" class="ms-docbutton"/> button it is possible to zoom the highlighted feature.
```

Comment:
The **Highlight Features** <img src="../img/button/highlight_features_icon.jpg" class="ms-docbutton"/> button allows to highlights on the map the layers features corresponding to the retrieved information in the clicked point.